client/my-sites/people/people-invites/index.jsx

```diff
@@ -31,32 +34,100 @@ class PeopleInvites extends React.PureComponent {
 		site: PropTypes.object.isRequired,
 	};
 
-	renderInvite = invite => {
-		const user = invite.user;
-
+	render() {
 		const { site } = this.props;
 
+		if ( ! site || ! site.ID ) {
+			return null;
+		}
+
 		return (
-			<PeopleListItem
-				key={ invite.key }
-				invite={ invite }
-				user={ user }
-				site={ site }
-				type="invite"
-				isSelectable={ false }
-			/>
+			<Main className="people-invites">
+				<QuerySiteInvites siteId={ site.ID } />
+				<SidebarNavigation />
+				<PeopleSectionNav filter="invites" site={ site } />
+				{ this.renderInvitesList() }
+			</Main>
 		);
-	};
+	}
+
+	renderInvitesList() {
+		const {
+			acceptedInvites,
+			pendingInvites,
+			totalInvitesFound,
+			requesting,
+			site,
+			translate,
+		} = this.props;
+
+		const hasAcceptedInvites = acceptedInvites && acceptedInvites.length > 0;
+		const acceptedInviteCount = ! requesting && hasAcceptedInvites ? acceptedInvites.length : null;
+
+		const hasPendingInvites = pendingInvites && pendingInvites.length > 0;
+		const pendingInviteCount = ! requesting && hasPendingInvites ? pendingInvites.length : null;
+
+		if ( ! hasPendingInvites && ! hasAcceptedInvites ) {
+			return requesting ? this.renderPlaceholder() : this.renderEmptyContent();
+		}
+
+		return (
+			<React.Fragment>
+				{ hasPendingInvites ? (
+					<div className="people-invites__pending">
+						<PeopleListSectionHeader
+							label={ translate( 'Pending' ) }
+							count={ pendingInviteCount }
+							site={ site }
+						/>
+						<Card>{ pendingInvites.map( this.renderInvite ) }</Card>
+					</div>
+				) : (
+					<div className="people-invites__pending">
+						{ this.renderInviteUsersAction( false ) }
+					</div>
+				) }
+
+				{ hasAcceptedInvites && (
+					<div className="people-invites__accepted">
+						<PeopleListSectionHeader
+							label={ translate( 'Accepted' ) }
+							count={ acceptedInviteCount }
+							// Excluding `site=` hides the "Invite user" link.
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Invite user', { context: 'button label'} )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/invites/split-pending-accepted)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
I think what's going on here is that this suggestion is a good one, but it's a bit broken because it's being reported against the wrong line of code.  I'm not sure who to ping about this.  @akirk maybe?

Reply:
@yoavf would be better :) This is the first time I've seen it; a8ci18n-bot just searched for the first occurrence of "Invite User," and, quite unusually, the string appeared earlier in the code at an unrelated spot. Thanks for flagging but I think in general it works good enough to keep it the way it is.